src/components/board.jsx

```diff
@@ -15,14 +15,14 @@ class ProgressView extends Component {
   state = {message: null, ticks: 0, max: 0};
 
   componentDidMount() {
-    const {progress} = this.props;
+    const { progress } = this.props;
```

Comment:
please undo all of these whitespace changes.  They do not belong in your PR.  Create an issue if you want to enforce a new style on the code in the repository.